### PR TITLE
chore: use embroider macro for skipping timeouts in tests

### PR DESCRIPTION
**Pull request by anehx**

### CODE REVIEW

File: packages/distribution/addon/components/inquiry-new-form.js [hunk 1]
```diff
@@ -63,10 +63,10 @@ export default class InquiryNewFormComponent extends Component {
   *updateSearch(e) {
     e.preventDefault();
 
-    const { environment } =
-      getOwner(this).resolveRegistration("config:environment");
-
-    if (environment !== "test") {
+    /* istanbul ignore next */
```
Comment (luytena) [inline packages/distribution/addon/components/inquiry-new-form.js:66]: What does `/* istanbul ignore next */` do?

File: packages/-ember-caluma/package.json [hunk 2]
```diff
@@ -14,6 +14,7 @@
   "devDependencies": {
     "@ember/jquery": "2.0.0",
     "@ember/optional-features": "2.0.0",
+    "@embroider/macros": "1.2.0",
```
Comment (luytena) [inline packages/-ember-caluma/package.json:17]: Why is the `^` not used in this `package.json` compared to the ones in the other packages?

Comment (anehx) [reply packages/-ember-caluma/package.json:17]: This is a dev dependency, not a dependency

File: packages/distribution/addon/components/inquiry-new-form.js (hunk 1 above)
Comment (anehx) [reply packages/distribution/addon/components/inquiry-new-form.js:66]: Ignores the next block for code coverage

**Review by luytena — APPROVED** [review]

**Pull request merged**